Credify/Credify/Objects/PostMessageModel.swift

```diff
@@ -26,4 +26,27 @@ internal enum ReceiveMessageHandler: String {
     case createUserCompleted
     case offerTransactionStatusChanged
     case actionClose
+    case bnplPaymentComplete
+}
+
+internal class StartBnplMessage: Codable {
+    let offers: [OfferData]
+    let profile: CredifyUserModel
+    let orderId: String
+    let completeBnplProviders: [Organization]
```

Comment:
`completeBnplProviders` is correct? Not `completedBnplProviders`?

Reply:
@ngo275 `completeBnplProviders` is correct.